packages/ember-runtime/lib/computed/reduce_computed_macros.js

```diff
@@ -278,6 +278,28 @@ export function mapBy(dependentKey, propertyKey) {
   hamster.get('remainingChores'); // [{name: 'write more unit tests', done: false}]
   ```
 
+  You can also use `@each` in your dependent key, the callback will still use the underlying array:
```

Comment:
🤔  I'm a bit concerned people will hurriedly read and start using `chores.@each` (for example) without the leaf property. Maybe `@each.property`? Maybe it's not a concern.